docs/contributing/contributing-setup.md

```diff
@@ -0,0 +1,162 @@
+--
+title: Contributing - Setting up your Dev Environment
+---
+
+### Prerequisites
+In order to contribute to Great Expectations, you will need the following:
+
+A GitHub account—this is sufficient if you only want to contribute to the documentation.
+
+If you want to contribute code, you will also need a working version of Git on your computer. Please refer to the Git setup instructions for your environment.
+
+We also recommend going through the SSH key setup process on GitHub for easier authentication.
+
+Fork and clone the repository
+
+1. Fork the Great Expectations repo
+
+Go to the Great Expectations repo on GitHub.
+
+Click the Fork button in the top right. This will make a copy of the repo in your own GitHub account.
+
+GitHub will take you to your forked version of the repository.
+
+2. Clone your fork
+
+Click the green Clone button and choose the SSH or HTTPS URL depending on your setup.
+
+````console
+Copy the URL and run git clone <url> in your local terminal.
+````
```

Comment:
could you have a look at the formatting here too?

Reply:
Looks ok in browser